content/topics/frameworks.md

```diff
@@ -10,23 +10,18 @@ intro = "When building a modern web-application you don't want to bother on how
 server = [
   "actix-web",
   "gotham",
-  "nickel",
   "rocket",
-  "tide",
```

Comment:
Why are we removing `tide` here?

Reply:
There has not been any updates for 3 years and as of May 2022 [the project is not under active development](https://github.com/http-rs/tide/discussions/888). The amount of downloads is still high, so it's not an easy decision to keep it or not. I don't mind either solution 👍🏽

Reply:
Let's stick to the criteria we agreed upon in https://github.com/rust-lang/arewewebyet/issues/127#issuecomment-2082811629
Based on that, I don't see a reason to remove it.